lib/glific/partners.ex

```diff
@@ -894,12 +894,20 @@ defmodule Glific.Partners do
   @spec credential_update_callback(Organization.t(), Credential.t(), String.t()) ::
           {:ok, any} | {:error, any}
   defp credential_update_callback(organization, credential, "bigquery") do
+    Caches.remove(organization.id, [{:provider_shortcode, "bigquery"}])
```

Comment:
Ohh just realise that is token cache key is too generic. 
Can you please make it 
**{:token, :provider_shortcode, provider_shortcode}**

Reply:
We need to change this in both functions. 

Also, can you move this function in partner.update_credentials function since it will be required for GCS and other things as well.

Reply:
@panka-ag made requested changes